admin.go

```diff
@@ -87,11 +86,8 @@ func AdminRebuildDBHandler(ctx *handlerContext, w http.ResponseWriter, r *http.R
 
 	//return /*
 	//db.Close()
-	os.Remove("./RebbleAppStore.db")
```

Comment:
Yeah that works. However, you'll need to do the same for other tables — at this point, I think it will be clearer if you just change the `CREATE TABLE` statements to add a `DROP TABLE` statement, like so:

```go
sqlStmt = `
		drop table authors;
		create table authors (
			id text not null primary key,
			name text
		);
		delete from authors;
	`
_, err = db.Exec(sqlStmt)
```